crates/sui-indexer/src/handlers/checkpoint_handler.rs

```diff
@@ -377,6 +382,9 @@ where
         let mut db_displays = BTreeMap::new();
         let mut db_indices = Vec::new();
 
+        let mut min_tx_sequence_number = checkpoint_summary.network_total_transactions;
```

Comment:
nit: tx_seq_num_iter.fold to get (min_cp, max_cp) ?

Reply:
since we're already looping through transactions once, i figured we could just plug into the existing logic instead of doing another pass, but not beholden to it